content/docs/reference/katib/v1alpha3/_index.md

```diff
@@ -0,0 +1,5 @@
++++
```

Comment:
I think we should get rid of this file, so that we avoid one extra level of clicks in the docs:
https://deploy-preview-1531--competent-brattain-de2d6d.netlify.com/docs/reference/katib/v1alpha3/

So, keep the directory as it indicates the version, but remove the index file.

Reply:
Thanks @sarahmaddox! I have removed content/docs/reference/katib/v1alpha3/_index.md.